cves/kernel/CVE-2020-12464.yml

```diff
@@ -55,7 +56,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This was a classic use-after-free issue which was caused by a race condition
```

Comment:
Might be helpful to add a brief description of what a race condition is

> A race condition occurs when two threads access a shared variable at the same time.

https://learn.microsoft.com/en-us/troubleshoot/developer/visualstudio/visual-basic/language-compilers/race-conditions-deadlocks

Reply:
Great point, I'll add that!

Reply:
great point, I'll add that!